services/gemma-local/main.py

```
@@ -0,0 +1,659 @@
+            json_body = await request.json()
+            
+            # Extract parameters from JSON body
+            audio = json_body.get("audio")
+            model = normalize_model_name(json_body.get("model", "gemma-2b"))
+            prompt = json_body.get("prompt")
+            response_format = json_body.get("response_format", "json")
+            temperature = json_body.get("temperature", 0.7)
+            language = json_body.get("language")
+            stream = json_body.get("stream", False)
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)